components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -1756,6 +1756,7 @@ type PortConfig struct {
 	Visibility  string  `json:"visibility,omitempty"`
 	Description string  `json:"description,omitempty"`
 	Name        string  `json:"name,omitempty"`
+	Sort        uint32  `json:"sort,omitempty"`
```

Comment:
Is it something only for internal usage of supervisor? I don’t think we should have such property here then. These are types for gitpod config.